### DISCUSSION TITLE
How to containerize wails application?

### DISCUSSION THREAD

**Discussion by leamandeep** [wailsapp/wails#3089] [Q&A]
I'm writing a desktop application using Wails. But the problem is how can I containerize my Wails source code.

**Comment by marcio199226** [comment]
I have played with https://github.com/mviereck/x11docker with some insights I was able to dockerize my app and execute it successfully

**Reply by leamandeep** [reply]
@marcio199226  my question is why do I need to do all the stuff? https://github.com/mviereck/x11docker. I'm just looking for a docker image. Btw I'm a beginner at building desktop applications.😅
